cpp/tests/strings/find_tests.cpp

```diff
@@ -63,6 +63,19 @@ TEST_F(StringsFindTest, Find)
     auto results = cudf::strings::rfind(strings_view, cudf::string_scalar(""));
     CUDF_TEST_EXPECT_COLUMNS_EQUAL(*results, expected);
   }
+  {
+    auto const targets = cudf::test::strings_column_wrapper({"l", "t", "", "x", "é", "o"});
```

Comment:
Can we test:
- multi-character targets
- a target string that is longer than the corresponding input string
- a null target string